javascript/app.js

```diff
@@ -0,0 +1,336 @@
+  "green pants",
+  "yellow knit hat",
+  "marshmallow peeps"
+];
+// Thom's closet is more complicated. Check out this nested data structure!!
+const thomsCloset = [
+  [
+    // These are Thom's shirts
+    "grey button-up",
+    "dark grey button-up",
+    "light blue button-up",
+    "blue button-up",
+  ],[
+    // These are Thom's pants
+    "grey jeans",
+    "jeans",
+    "PJs"
+  ],[
+    // Thom's accessories
+    "wool mittens",
+    "wool scarf",
+    "raybans"
+  ]
+];
+
+// Alien Attire
+// 1. Kristyn's left shoe has traveled through time and space and turned up in Thom's accessories drawer! Remove Kristyn's shoe from the array and save it to the variable kristynsShoe. Use that variable to add Kristyn's lost shoe to Thom's accessories array.
+
+// take the value (its an object)
+let kristynsShoe = kristynsCloset.splice(0, 1);
+// turn it into a string;
+kristynsShoe = kristynsShoe[0];
+// add to thom's accessories
+thomsCloset[2].push(kristynsShoe);
+// print to make sure
+console.log(kristynsCloset, thomsCloset);
+
+// Dress 'em Up
+// 1. Modify your code to put together 3 separate outfits for Kristyn and Thom. Put the output in a sentence to tell us what we'll be wearing. Mix and match!
+
+// KRISTYN
+// outfit combo declaration
+let outfitComboKristyn = [];
+// three possible outfits
+for (let h = 0; h < 3; h++){
+	// pick two pieces for one outfit
+	for (let i = 0; i < 2; i++) {
+		// at random
+		let randomValue = Math.floor(Math.random() * (kristynsCloset.length - outfitComboKristyn.length));
+		// test randomValue
+		console.log("Random value is " + randomValue);
+		// remove item from closet and put it into outfitComboKristyn
+		outfitComboKristyn.push(kristynsCloset.splice(randomValue, 1));
+	}
+	// to keep track of array
+	// console.log(outfitComboKristyn);
+	console.log(`Today Kristyn might wear ${outfitComboKristyn[0]} and ${outfitComboKristyn[1]}.`);
+	outfitComboKristyn = [];
+}
+
+// THOM
+// outfit combo declaration
+let outfitComboThom = [];
+// three possible outfits
+for (let h = 0; h < 3; h++){
+	// pick two pieces for one outfit
+	for (let i = 0; i < 2; i++) {
+		// random number to choose which drawer to take from
+		let randomValue = Math.floor(Math.random() * 2);
+		// test randomValue
+		console.log("Random value is " + randomValue);
+		// random number to choose which ITEM to take in drawer
+		let randomValue2 = Math.floor(Math.random() * (thomsCloset[randomValue].length - 1));
+		// test randomValue
+		console.log("Random value2 is " + randomValue2);
+		// remove item from drawer and put it into outfitComboThom
+		//vv SOMETHING WRONG with array length!
+		//outfitComboThom.push(thomsCloset[randomValue2].splice(randomValue2, 1));
+	}
+	// to keep track of array
+	// console.log(outfitComboThom);
+	console.log(`Today Thom might wear ${outfitComboThom[0]} and ${outfitComboThom[1]}.`);
+	outfitComboThom = [];
+	console.log(thomsCloset);
+}
+// ^^Something's missing problem with array lengths I think
```

Comment:
yes i believe Thom's clothes are undefined and if he leaves the house he'll get arrested for indecent exposure 😕🙃